pydoctor/epydoc2stan.py

```diff
@@ -448,7 +475,8 @@ def _handle_param_not_found(self, name: str, field: Field) -> None:
             if name in source.constructor_params:
                 # Constructor parameters can be documented on the class.
                 return
-        field.report('Documented parameter "%s" does not exist' % (name,))
+        if not name.startswith('*'):
```

Comment:
I guess that this is why you wanted to add the asterix ... I am not happy with the result, but you are the one implementing it...so if this is what you want, it's ok :)

Reply:
Can you explain what you mean exactly ? 
I'm happy with the result on my end, but I'de like to know what's wrong in your opinion.

Thanks

Reply:
Here, we have to keep the ``.startswith('*')`` because, now we set the name with stars in ``_handle_param_name``, and that's the name that we have here.

Reply:
I think I've fixed the issue you raised.